includes/templates/class-amp-content.php

```diff
@@ -198,8 +202,9 @@ private function register_embed_handlers( $embed_handler_classes ) {
 	 */
 	private function unregister_embed_handlers( $embed_handlers ) {
 		foreach ( $embed_handlers as $embed_handler ) {
-			$this->add_scripts( $embed_handler->get_scripts() ); // @todo Why add_scripts here? Shouldn't it be array_diff()?
-			$embed_handler->unregister_embed();
+			if ( $embed_handler instanceof Registerable ) {
```

Comment:
Needed for backwards-compatibility:

```suggestion
			if ( $embed_handler instanceof Registerable || method_exists( $embed_handler, 'unregister_embed' ) ) {
```